src/applications/gi/containers/GiBillApp.jsx

```diff
@@ -85,7 +86,10 @@ export class GiBillApp extends React.Component {
     if (constants.inProgress) {
       content = <LoadingIndicator message="Loading..." />;
     } else {
-      content = this.props.children;
+      content =
+        constants.statusCode === (503 || 500 || 504)
```

Comment:
Should this be replaced with a function in `src/applications/gi/utils/helpers.js` and then replace other usages with the function?

Reply:
its unique to each page